manifests/init.pp

```diff
@@ -132,6 +133,7 @@
   Optional[Enum['yes','no']] $cvmfs_follow_redirects                 = undef,
   Boolean $cvmfs_instrument_fuse                                     = false,
   Boolean $cvmfs_repo_list                                           = true,
+  Optional[Cvmfs::IPFamily] $cvmfs_ipfamily_prefer                   = undef,
```

Comment:
Don't thine we need a custom type as we already have much longer types and I doubt we will use this elsewhere.
So variant good but just stick it here.

Reply:
Sounds good, I have just updated the code with the proposed change.